functionary/core/models/user_role.py

```diff
@@ -58,6 +60,7 @@ class EnvironmentUserRole(UserRole):
         on_delete=models.CASCADE,
         related_name="user_roles",
     )
+    inherited = models.BooleanField(default=True)
```

Comment:
This isn't something that we should be keeping track of, the user doesn't get to see what their permissions are, only the environments that they have access to via the environment chooser.

Reply:
Only admins for an environment are able to see other users roles, which is for the sake of user management. 

The reason the inheritance is tracked is so that admins for the environments can distinguish between which users have inherited their access to the environment by being on the team, versus users who have been added to the environment from a different team.